packages/zoltan2/src/input/Zoltan2_InputTraits.hpp

```diff
@@ -411,6 +429,7 @@ struct InputTraits<Tpetra::Vector<Scalar,LocalOrdinal,GlobalOrdinal,Node> >
   typedef Scalar        scalar_t;
   typedef LocalOrdinal  lno_t;
   typedef GlobalOrdinal gno_t;
+  typedef LocalOrdinal offset_t;
```

Comment:
If possible, let's make offset_t be size_t for all the Xpetra, Tpetra, Epetra adapters.